app/waf/tsconfig.json

```diff
@@ -1,4 +1,10 @@
 {
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
```

Comment:
I not quite sure about what will be impacted, if we specify 'ES5' or 'ES6' as the target language. Which one will be best for us?

Reply:
It seems not abvious to us since we have no webpage except /explorer. 
The 2 settings is related to webbrowser's supportability. See the link: https://blog.angularindepth.com/configuring-typescript-compiler-a84ed8f87e3, search `Transpiling`. 
To us, select 'ES5' is most stable. ES6 is somehow too new. ES3 is somehow too old(it's default).